src/components/Person/Person.jsx

```diff
@@ -1 +1,46 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Person = ({
+  person: {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  },
+}) => {
+  const wifeOrHusband = sex === 'm'
+    ? ('wife')
+    : ('husband');
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+      {age && (
+        <p className="Person__age">{`I am ${age}`}</p>
+      )}
+      <p className="Person__partner">
+        {isMarried
+          ? (`${partnerName} is my ${wifeOrHusband}`)
+          : ('I am not married')
+        }
+      </p>
+    </section>
+  );
+};
+
+Person.defaultProps = {
```

Comment:
you don't need default values at all. `!undefined === true` and `undefined === false`